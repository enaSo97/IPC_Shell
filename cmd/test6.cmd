ls *.h dish_main.c
