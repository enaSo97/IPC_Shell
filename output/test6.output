dish_main.c
dish_run.h
dish_tokenize.h
in while
found the wild card
new arr sz: 3
globbing index 1
Child(<2716>) exited -- Success (<0>)
