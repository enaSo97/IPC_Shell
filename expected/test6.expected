dish_main.c
dish_run.h
dish_tokenize.h
